src/main/java/at/hannibal2/skyhanni/data/PetAPI.kt

```diff
@@ -35,31 +190,438 @@ object PetAPI {
         "➡",
     )
 
+    @Deprecated(message = "use PetAPI.inPetMenu")
     fun isPetMenu(inventoryTitle: String): Boolean = petMenuPattern.matches(inventoryTitle)
 
     // Contains color code + name and for older SkyHanni users maybe also the pet level
+    @Deprecated(message = "use PetAPI.pet.name")
     var currentPet: String?
         get() = ProfileStorageData.profileSpecific?.currentPet?.takeIf { it.isNotEmpty() }
         set(value) {
             ProfileStorageData.profileSpecific?.currentPet = value
         }
 
+    @Deprecated(message = "use PetAPI.pet.rawPetName",
+        replaceWith = ReplaceWith("pet.name.contains(petName) ?: false", "at.hannibal2.skyhanni.data.PetAPI.pet")
+    )
     fun isCurrentPet(petName: String): Boolean = currentPet?.contains(petName) ?: false
```

Comment:
i feel like its better to modify this function to use the new data than to make people use `pet.name.contains(petName) ?: false`